site/content/blogs/release-v123.ja.md

```diff
@@ -0,0 +1,209 @@
+- **ルートファイルシステムへのコンテナアクセスを読み取り専用に制限:** [詳細はこちらを確認してください。](../docs/manifest/lb-web-service.ja.md#storage-readonlyfs) ([#4062](https://github.com/aws/copilot-cli/pull/4062)).
+- **ALB の HTTPS レイヤーに対する SSL ポリシーを設定します:** [詳細はこちらを確認してください。](../docs/manifest/environment.ja.md#http-public-sslpolicy) ([#4099](https://github.com/aws/copilot-cli/pull/4099)).
+- **ALBに対する ソース IP によるアクセス制限**: [詳細はこちらを確認してください。](../docs/manifest/environment.ja.md#http-public-ingress-source-ips) ([#4103](https://github.com/aws/copilot-cli/pull/4103)).
+
+
+???+ note "AWS Copilot とは?"
+
+    AWS Copilot CLI は AWS 上でプロダクションレディなコンテナ化されたアプリケーションのビルド、リリース、そして運用のためのツールです。
+    開発のスタートからステージング環境へのプッシュ、本番環境へのリリースまで、Copilot はアプリケーション開発ライフサイクル全体の管理を容易にします。
+    Copilot の基礎となるのは、 AWS CloudFormation です。CloudFormation により、インフラストラクチャを 1 回の操作でコードとしてプロビジョニングできます。
+    Copilot は、さまざまなタイプのマイクロサービスの作成と運用の為に、事前定義された CloudFormation テンプレートと、ユーザーフレンドリーなワークフローを提供します。
+    デプロイメントスクリプトを記述する代わりに、アプリケーションの開発に集中できます。
+
+    より詳細な AWS Copilot の紹介については、[Overview](../docs/concepts/overview.ja.md) を確認してください。
+
+<a id="app-runner-private-services"></a>
+
+## App Runner プライベートサービス
+Copilot を使って App Runner プライベートサービスを作成出来ます。Request-Driven Web Service の Manifest を更新し、
+```yaml
+http:
+  private: true
+```
+デプロイするだけです！　その Service は、Copilot Envrionment 内の他の Service からのみ到達できます。
+その舞台裏では、Copilot が Envrionment 内の全てのプライベートな Service で共有される APP Runnner の VPC エンドポイントを作成しています。
+既存の App Runner VPC エンドポイントがある場合、Manifest に次の様な設定をして、インポート出来ます。
+```yaml
+http:
+  private:
+    endpoint: vpce-12345
+```
+デフォルトでは、プライベートサービスはインターネットにのみトラフィックを送る事ができます。
+Environment 内にトラフィックを送りたい場合は、Manifest に [`network.vpc.placement: 'private'`](../docs/manifest/rd-web-service.ja.md#network-vpc-placement) と設定します。
+
+<a id="support-aurora-serverless-v2-in-storage-init"></a>
+
+## [`storage init`] (../docs/commands/storage-init.ja.md)で Aurora Serverless v2 をサポート
+[Aurora Serverless v2 は 今年の初めに一般利用となりました。](https://aws.amazon.com/about-aws/whats-new/2022/04/amazon-aurora-serverless-v2/)
+現在、Copilot のストレージオプションとしてサポートされています。
+
+以前は、次のコマンドを実行し、
+```console
+$ copilot storage init --storage-type Aurora
+``` 
+v1 クラスター用の Addon テンプレートを作成しました。現在は、**このコマンドはデフォルトで v2用のテンプレートを作成します。**
+v1 テンプレートを作成したい場合、`copilot storage init --storage-type Aurora --serverless-version v1` というコマンドを実行します。
+
+より詳しく知りたい場合は、[the doc for `storage init`](../docs/commands/storage-init.ja.md)を確認してください！
+
+<a id="move-misplaced-http-fields-in-environment-manifest-backward-compatible"></a>
+
+## Environment Manifestにおける誤った `http` フィールドの移動（後方互換性あり！）
+
+[Copilot v1.23.0](https://github.com/aws/copilot-cli/releases/tag/v1.23.0) では、 Environment Manifest における `http` フィールド下の階層を修正しました。
+
+### 何が修正されるのか、それは何故なのか？
+[Copilot v1.20.0](../blogs/release-v120.en.md)では、Environment Manifest をリリースし、infrastructure as code の利点を Environment にも取り込みました。当時の `http` フィールド階層は次の様な形です。
```

Comment:
Could you reference the release blog in Japanese(`blogs/release-v120.ja.md`) here?

Reply:
Yes. I will fix it.